src/app/story/directives/basic.component.ts

```diff
@@ -36,16 +36,21 @@ import { HalDoc, ModalService, TabService } from 'ngx-prx-styleguide';
       <hr/>
 
       <prx-fancy-field required label="Audio Files">
-        <prx-select *ngIf="versionTemplateOptions" class="file-select"  placeholder="Select Templates..."
+        
+        <prx-select
```

Comment:
Pre-existing styleguide thing ... but what's with the 2 different delete X's on the dropdown?

![](http://cav.is/i/gupi9.jpg)

Reply:
Because switching from multiple select to single select changes the underlying data source from an `any[]` to an `any` this is a multiple select with a maximum of one item selectable. The X on the right clears all items, while the X on the left clears that specific item. In this particular case, there's no difference.

Awkward, not sure it's worth working around? We could hide the right-hand X for a `prx-select` with `maxSelectedItems === 1`

Reply:
+1 for this 
> We could hide the right-hand X for a prx-select with maxSelectedItems === 1

Reply:
Not even sure where to put this, as most changes are in Styleguide, but this is what I ended up with:

![image](https://giant.gfycat.com/HauntingVastInvisiblerail.gif)

Reply:
Great. I like it!